src/lua/BabblerBomb/Gorge.lua

```diff
@@ -0,0 +1,16 @@
+Script.Load("lua/BabblerBomb/BabblerBomb.lua")
+Script.Load("lua/BabblerBomb/BabblerBombAbility.lua")
+
+if Server then
+	
+	if kCombatVersion then
+		function Gorge:GetTierThreeTechId()
+			return kTechId.BabblerBombAbility
+		end
+	else

```

Comment:
What's the purpose of `kCombatVersion`? I feel like if a combat mod wants to adjust the level of this new tech, it should be adjusted in the combat mod.